src/lib/sidebar/scheme_data.ts

```diff
@@ -22,6 +23,10 @@ export function backfill(json: Scheme) {
     // Always overwrite IDs, and follow what newFeatureId requires.
     f.id = idCounter++;
   }
+  const uuid = uuidv4();
+  json.schemes = json.schemes || {
+    uuid: getEmptySchemeData(uuid),
```

Comment:
I'm fairly sure this creates a dictionary literally with the key "uuid", not the variable. How about
```
if (!json.schemes) {
  json.schemes = {};
  json.schemes[uuid] = {
    scheme_reference: uuid,
  };
}
```

Reply:
You are absolutely right: I have encountered and fixed this during fixing the tests 😅